site/content/en/latest/tasks/operations/standalone-deployment-mode.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "Standalone Deployment Mode"
+---
+
+{{% alert title="Notice" color="warning" %}}
+
+Standalone mode is an experimental feature, please **DO NOT** use it in production.
+
+{{% /alert %}}
+
+Envoy Gateway also supports running in standalone mode. In this mode, you can configure the
+Envoy Gateway to use different file provider and infrastructure provider combinations.
+
+- The file provider will configure the Envoy Gateway where to get all gateway-api resources.
+- The infrastructure provider will configure the Envoy Gateway how to deploy the data-plane components.
+
+Currently, the types supported by file provider are:
+
+- **File**, retrieve gateway-api resources from file system.
+
+The infrastructure provider supports the following types:
+
+- **Host**, run data-plane component as a host process.
+
+## Quick Start
+
+In this quick-start, we will run Envoy Gateway in standalone mode with **File** type file provider
+and **Host** type infrastructure provider.
+
+### Prerequisites
+
+Create a local directory just for testing:
+
+```shell
+mkdir -p /tmp/envoy-gateway-test
+```
+
+Compile Envoy Gateway binary from project by using command:
+
+```shell
+make build
```

Comment:
currently, we let users to build envoy-gateway binary from scratch. should we include this binary in release ?

Reply:
Event in standalone mode, we should use docker instead of pure binary?

Reply:
For now, I think it's fine to use docker to spin up the EG in the documentation.

However, we should include the binaries to the release assets and update the doc to add the binary option. This can be addressed is a follow-up PR.

Reply:
we shouldnt be use `make` command here, instead either use GH releases assets https://github.com/envoyproxy/gateway/releases/tag/v1.2.0-rc.1 (if `envoy-gateway` is not present, we should consider pushing it), or Docker Image, my vote is for GH releases

Reply:
if you build binary, you will need to support all the platforms(e.g. macos, windows), TBH, I think there's a kind of waste?

Reply:
there are many production environments/server thats are running a server with a manager like systemd that have no docker installed

Reply:
similar to envoyproxy https://github.com/envoyproxy/envoy/releases/tag/v1.32.0 , imo `linux-x86_64` is good enough for now

Reply:
like @zhaohuabing 's idea. we should include eg binary in release, and maybe just for linux only, this can be done as a follow-up.

but for now, using docker to start eg sgtm.

Reply:
raised https://github.com/envoyproxy/gateway/issues/4566 to track the GH release work

Reply:
the binary is now part of the artifacts, hope this unblocks you @shawnh2